src/main/frontend/handler/user.cljs

```diff
@@ -151,20 +171,38 @@
 
 
 ;;; refresh tokens loop
-(def stop-refresh false)
+(def stop-ch (chan (async/dropping-buffer 1)))
+(def refresh-now-ch (chan))
+(def refresh-finish-ch (chan (async/sliding-buffer 1)))
+
+
 (defn refresh-tokens-loop []
```

Comment:
This might be something we can remove later since this PR introduces `<ensure-id&access-token` before any request to the server.

Reply:
Also, this commit introduces another two channels, it could be nice if we could maintain only a few channels and go-loop handlers.